backend/packages/Upgrade/src/api/controllers/ExperimentClientController.ts

```diff
@@ -35,12 +35,7 @@ import { env } from '../../env';
 interface IExperimentAssignment {
   expId: string;
   expPoint: string;
-  assignedCondition: {
-    conditionCode: string;
-    conditionAlias: string;
-    experimentId: string;
-    id: string;
-  };
+  assignedCondition: object;
```

Comment:
Why have you changed this to a generic object type?

Reply:
Why have you changed this to a generic object type?

Reply:
code updated